xarray/core/common.py

```diff
@@ -474,6 +474,32 @@ def rolling(self, min_periods=None, center=False, **windows):
         Returns
         -------
         rolling : type of input argument
+        
+        Examples
+        --------
+        Create rolling seasonal average of monthly data e.g. DJF, JFM, ..., SON:
+        
+        >>> da = xr.DataArray(np.linspace(0,11,num=12), 
+                              coords=[pd.date_range('15/12/1999',
```

Comment:
Use `...` to continue a line (notice that this is what the `python` interpreter does).